js/main.js

```diff
@@ -0,0 +1,33 @@
+/*
+Функция, возвращающая случайное целое число из переданного диапазона включительно.
+
+Пример использования функции:
+  имя_функции(от, до);
+  Результат: целое число из диапазона "от...до"
+
+Источник:https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+*/
+
+const getRandom = (from, to) => {
+  if (from > to) {
+    return new Error('InvalidOrderOfArgument');
+  }
+  const min = Math.ceil(from);
+  const max = Math.floor(to);
+  return Math.floor(Math.random() * (max - min + 1) + min);
+};
+
+/*
+Функция для проверки максимальной длины строки.
+
+Пример использования функции:
+имя_функции(проверяемая_строка, максимальная_длина);
+  Результат:
+    true, если строка проходит по длине,
+    false — если не проходит
+
+*/
+
+const checkStringLenght = (string, maxLenght) => string.lenght <= maxLenght;
```

Comment:
Welcome to javascript! Here you're trying to access `string.lenght` - there is a typo in word `length` (literally everywhere in this code :) ) which would result in, probably, error in other langs, but there you'll always get `undefined` as a result. Then, when you are trying to compare `undefined` and `number` the first one will convert to `NaN`. As a result you'll always return `false`

Reply:
Thank al lot :)